src/compositor/lipstickcompositor.cpp

```diff
@@ -732,3 +737,35 @@ void LipstickCompositor::readContent()
 {
     m_recorder->recordFrame(this);
 }
+
+void LipstickCompositor::timerEvent(QTimerEvent *e)
+{
+    if (isVisible()) {
+        stopRendering();
```

Comment:
We need to keep drawing to give a chance to the clients to have their buffers released, after they receive the onscreen_visibility event.
As for the touch, afaik mce grabs it indeed. @spiiroin can you confirm?